sdk/core/azure-core/tests/async_tests/test_authentication_async.py

```diff
@@ -37,17 +39,18 @@ async def get_token(_):
     policies = [AsyncBearerTokenCredentialPolicy(fake_credential, "scope"), Mock(send=verify_authorization_header)]
     pipeline = AsyncPipeline(transport=Mock(), policies=policies)
 
-    await pipeline.run(HttpRequest("GET", "https://spam.eggs"), context=None)
+    await pipeline.run(http_request("GET", "https://spam.eggs"), context=None)
     assert get_token_calls == 1
 
-    await pipeline.run(HttpRequest("GET", "https://spam.eggs"), context=None)
+    await pipeline.run(http_request("GET", "https://spam.eggs"), context=None)
     # Didn't need a new token
     assert get_token_calls == 1
 
 
-async def test_bearer_policy_send():
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
```

Comment:
I parametrized all of these tests to run twice: one with the old requests, and once with the new reqests